### ISSUE TITLE
Indexer shouldn't throw exception if vertex by label not found

### ISSUE THREAD

**Issue by szdavid92** [FTSRG/ingraph#186]
NoSuchElementException is thrown when name not found
```
(.verticesByLabel (Indexer. ) "name")
NoSuchElementException key not found: name  scala.collection.MapLike$class.default (MapLike.scala:228)
```

**Comment by wafle** [comment]
Yes, the default Scala behavior is forwarded. Why is this an issue?

**Comment by szdavid92** [comment]
I think using exceptions for this is just clumsy. This is not an exceptional situation. The map may or may not have an item.

**Comment by szdavid92** [comment]
Resolved by 94ee18a
